css-sass/index.html

```diff
@@ -0,0 +1,321 @@
+        </div>
+        <div class="position-relative text-center">
+          <img src="../assets/images/image4.jpeg" alt="image4" class="w-8 h-8">
+          <p class="text-white font-medium text-base position-absolute top-75 left-50 transform-X-50">Merchandizing</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="flex flex-col items-center gap-4 mt-5 w-80 m-inline">
+    <h5 class="text-center font-medium text-lg leading-tight tracking-tight w-60 text-center">Trabajamos día a día junto
+      a empresas que confían en nosotros
+    </h5>
+    <div class="flex flex-wrap justify-center w-80 gap-2">
+      <img src="../assets/images/logo1.jpeg" alt="logo1" class="w-5 h-1">
+      <img src="../assets/images/logo2.jpeg" alt="logo2" class="w-5 h-1">
+      <img src="../assets/images/logo3.jpeg" alt="logo3" class="w-5 h-1">
+      <img src="../assets/images/logo4.jpeg" alt="logo4" class="w-5 h-1">
+      <img src="../assets/images/logo5.jpeg" alt="logo5" class="w-5 h-1">
+      <img src="../assets/images/logo6.jpeg" alt="logo6" class="w-5 h-1">
+      <img src="../assets/images/logo7.jpeg" alt="logo7" class="w-5 h-1">
+      <img src="../assets/images/logo8.jpeg" alt="logo8" class="w-5 h-1">
+
+      <img src="../assets/images/logo9.jpeg" alt="logo9" class="w-5 h-1">
+      <img src="../assets/images/logo10.jpeg" alt="logo10" class="w-5 h-1">
+      <img src="../assets/images/logo11.jpeg" alt="logo11" class="w-5 h-1">
+      <img src="../assets/images/logo12.jpeg" alt="logo12" class="w-5 h-1">
+      <img src="../assets/images/logo13.jpeg" alt="logo13" class="w-5 h-1">
+      <img src="../assets/images/logo14.jpeg" alt="logo14" class="w-5 h-1">
+      <img src="../assets/images/logo15.jpeg" alt="logo15" class="w-5 h-1">
+      <img src="../assets/images/logo16.jpeg" alt="logo16" class="w-5 h-1">
+    </div>
+  </section>
+
+  <section class="flex flex-col mt-5 gap-7 w-80 m-inline">
+    <div class="flex items-center gap-6">
+      <div class="flex flex-col gap-3">
+        <h3 class="font-semibold	 text-2xl leading-tight text-transparent-bgclip">Ahorrá tiempo</h3>
+        <h4 class="font-medium text-xl leading-tight">Planificá tu año en base a tu presupuesto y programá tus acciones.
+          Línea directa entre proveedores y empresas
+        </h4>
+      </div>
+      <img src="../assets/images/highlight1.jpeg" alt="highlight1" class="w-10 h-9">
+    </div>
+    <div class="flex items-center gap-6">
+      <img src="../assets/images/highlight2.jpeg" alt="highlight2" class="w-10 h-9">
+      <div class="flex flex-col gap-3">
+        <h3 class="font-semibold	 text-2xl leading-tight text-transparent-bgclip">Comprendé a tu equipo</h3>
+        <h4 class="font-medium text-xl leading-tight">Obtené feedback para mejorar tus acciones en cada etapa.</h4>
+      </div>
+    </div>
+  </section>
+
+  <section class="flex flex-col items-center mt-5 w-80 m-inline">
```

Comment:
the class name `m-inline` is not an indicator of what we are trying to achieve with this class. margin-inline is the property and auto the value we are using, so i think that m-inline-auto makes mores sense,